src/client/Microsoft.Identity.Client/Platforms/iOS/iOSBroker.cs

```diff
@@ -242,6 +258,32 @@ private bool ValidateBrokerResponseNonceWithRequestNonce(Dictionary<string, stri
             return true;
         }
 
+        private void WriteBrokerApplicationTokenToKeychain(string clientId, string applicationToken)
+        {
+            iOSTokenCacheAccessor iOSTokenCacheAccessor = new iOSTokenCacheAccessor();
+
+            SecStatusCode secStatusCode = iOSTokenCacheAccessor.SaveBrokerApplicationToken(clientId, applicationToken);
```

Comment:
>SecStatusCode secStatusCode = iOSTokenCacheAccessor.SaveBrokerApplicationToken(clientId, applicationToken); [](start = 12, length = 107)

could this result in an exception or will a failure result always be in SecStatusCode? if not the log statement may not be executed